system/install-service.sh.in

```diff
@@ -265,8 +265,14 @@ install_systemd_service() {
         warning "Failed to reload systemd unit files."
     fi
 
+    if [ "${ENABLE}" = "enable" ]; then
+      if ! systemctl unmask netdata; then
+        warning "Failed to unmask Netdata service."
+      fi
+    fi
+
```

Comment:
The problem is that the _distro_ may have masked it, and not handled things correctly. Debian and Ubuntu mask services on package uninstall for complex reasons that have to do with ensuring consistency between systemd services and traditional init.d scripts.

That said, given the above info from @thiagoftsm, we should not actually need this at all, because the issue is not in the installation of the systemd service, it’s some problem in the auto-detection code.